proxy/ha_proxy.go

```diff
@@ -227,6 +229,14 @@ func (m HaProxy) getConfigData() ConfigData {
     option  dontlognull
     option  dontlog-normal`
 	}
+
+	default_ports := []string{"80", fmt.Sprintf("443%s", d.CertsString)}
```

Comment:
`DEFAULT_PORTS` in Dockerfile cannot have certs since they can be added dynamically throughout the lifetime of the proxy. I meant the second case. `:ssl` can be replaced with contents of `CertsString`. I just made a commit to the `sni2` branch with the change I had in mind. Please take a look at https://github.com/vfarcic/docker-flow-proxy/commit/141cf557d458a2e3d6a4a73e882807daf9afbd7a.

Something strange is happening with `go fmt ./...`. Seems that the formatting it applies is not the same on my machine as on yours :(. I need to figure out why is that happening. Anyways, the files from the above mentioned commit that contain the changes are *Dockerfile*, *ha_proxy.go*, and *haproxy_test.go*.

Please let me know what you think about the commit.